lib/ansible/module_utils/facts/virtual/linux.py

```diff
@@ -186,7 +186,7 @@ def get_virtual_facts(self):
 
         sys_vendor = get_file_content('/sys/devices/virtual/dmi/id/sys_vendor')
 
-        KVM_SYS_VENDORS = ('QEMU', 'oVirt', 'Amazon EC2', 'DigitalOcean', 'Google', 'Scaleway', 'Nutanix')
+        KVM_SYS_VENDORS = ('QEMU', 'oVirt', 'Red Hat', 'Amazon EC2', 'DigitalOcean', 'Google', 'Scaleway', 'Nutanix')
```

Comment:
Could you please add a change log entry for this change? Thanks.

Reply:
Please remove that line, for RHV detected we have code above, oVirt should be detected as kvm guest